### ISSUE TITLE
[feature request] Support exclude-tags parameter

### ISSUE THREAD

**Issue by dwcramer** [minivan/middleman-spellcheck#8]
My site has a lot of source code-like things like <code>PredicateFactory</code> that are being picked up by the spell checker. I would like to be able to do:

```
activate :spellcheck, exclude-tags: [:code, :pre]
```

to cause the spell checking to avoid checking these elements that cause lots of spurious errors.

**Comment by wkoszek** [comment]
@dwcramer I second that. With the pull #18 :

https://github.com/minivan/middleman-spellcheck/pull/18/files

I'd also want to remove URL checking.

**Comment by zealot128** [comment]
With recent commit, code, style and script tags are ignored by default:

https://github.com/minivan/middleman-spellcheck/blob/master/lib/middleman-spellcheck/extension.rb#L49

**Comment by wkoszek** [comment]
This looks great. Can we bump a number after this change and release it 
as 8.1 or something?

It'd be great to have "tag_exclude_override" so that we could 
override/add some tags to the already existing list.

W.

**Comment by wkoszek** [comment]
@zealot128 I think this issue can be closed?
